dashboard/views/users.ejs

```diff
@@ -110,7 +110,7 @@
                                               <% if(data.users[i].language =='pl') {%> <td>Polish</td> <%} %>
                                               <% if(data.users[i].language =='ibo') {%> <td>Igbo</td> <%} %>
                                               <% if(data.users[i].language =='yor') {%> <td>Yoruba</td> <%} %>
-                                              <td><%= data.users[i].role %></td>
+                                              <% if(query.role != 'admin') {%> <td data-l10n-id="student">Student</td><%} else {%> <td data-l10n-id="admin">Admin</td> <%} %>
```

Comment:
use something like this instead
`<td data-l10n-id="{{query.role.toLowerCase()}}">Student</td>`